web/modules/mof/src/Form/ModelForm.php

```diff
@@ -158,25 +309,88 @@ public function form(array $form, FormStateInterface $form_state): array {
             'class' => ['component-license-wrapper'],
             'id' => "component-{$cid}",
           ],
-          'label' => [
-            '#type' => 'html_tag',
-            '#tag' => 'span',
-            '#value' => $component->name,
+          'included' => [
+            '#type' => 'checkbox',
+            '#title' => $component->name,
+            '#default_value' => in_array($cid, $model_components) ? $cid : false,
+            '#return_value' => $cid,
+            '#parents' => ['component', $cid],
+          ],
+          'global' => [
+            '#type' => 'checkbox',
+            '#parents' => ['global', $cid],
+            '#title' => $this->t('Does this component use the global license?'),
```

Comment:
I think given that it's a checkbox it'd be better not to make it a question but something like:
```suggestion
            '#title' => $this->t('This component uses the global license.'),
```
Or if you want to make it more self descriptive:
```suggestion
            '#title' => $this->t('Check if this component uses the global license.'),
```

Reply:
"Check if..." sounds better to me.